.github/workflows/centos_7.yml

```diff
@@ -2,18 +2,23 @@ name: centos_7
 
 on:
   push:
+    branches:
+      - 'master'
+      - '[0-9].[0-9]+'
+      - '**-full-ci'
+    tags:
+      - '**'
   pull_request:
+    types: [opened, reopened, synchronize, labeled]
   repository_dispatch:
     types: [backend_automation]
   workflow_dispatch:
 
 jobs:
   centos_7:
-    # We want to run on external PRs, but not on our own internal PRs
-    # as they'll be run by the push to the branch.
-    if: ( github.event_name == 'push' ||
-        github.event.pull_request.head.repo.full_name != github.repository ) &&
-        ! endsWith(github.ref, '-notest')
```

Comment:
Do you decided to don't bother about `-notest`? It was convenient sometimes.

Reply:
I see now, it is needed only for small set of branches. Sorry for noise.

Reply:
This workflow is run only for `**-full-ci` branches (see the branch filter above). For all other branches (including `**-notest`), it won't be triggered.